docs/reset-password.md

```diff
@@ -0,0 +1,54 @@
+# Reset TechPass password
+
+If you are a vendor and if your TechPass domain is ```techpass.gov.sg```, you may reset your TechPass password according to the [Azure Password Policy](https://learn.microsoft.com/en-us/azure/active-directory/authentication/concept-sspr-policy#administrator-password-policy-differences) or if your TechPass account gets locked.
+
+>**Notes for public officers**
+>
+>- If you are a public officer, and if your TechPass account gets locked after continuous unsuccessful login attempts, reset the password of your GSIB device by following the WOG password policies. 
+>
+>- If there are any issues, contact your Agency Facility Management (AFM).
+
+**To reset TechPass password**
+
+1. Go to [reset password](https://passwordreset.microsoftonline.com/) and  enter your TechPass user name.
+
+> **Note**
+>- If the email address you specified while requesting for a TechPass account, is a Microsoft account, you can either enter that email address or your TechPass user name.
+>- TechPass user name is sent to you in the TechPass onboarding email.
+
+2. Enter the characters in the picture or the words in the audio.
```

Comment:
I'm assuming `audio` is a typo.

Reply:
@JLeeChan , nope :-),  that is what is displayed on the azure UI

Reply:
oh.. ok then